packages/paste-core/components/data-grid/src/DataGrid.tsx

```diff
@@ -24,133 +24,137 @@ export interface DataGridProps extends TableProps {
  * DataGrid wrapper component.
  *
  * @param {string} aria-label - for screen readers
+ * @param {boolean} striped - zebra striping on table rows
  * @param {string} element - customization element
  */
-export const DataGrid = React.forwardRef<HTMLTableElement, DataGridProps>(({element = 'DATA_GRID', ...props}, ref) => {
-  const dataGridId = `data-grid-${useUID()}`;
-  const lastFocusedElement = React.useRef<Element | null>(null);
-  const compositeState = useCompositeState({unstable_virtual: false});
-  const [actionable, setActionable] = React.useState<boolean>(false);
+export const DataGrid = React.forwardRef<HTMLTableElement, DataGridProps>(
```

Comment:
hide whistespace changes when viewing this file
![image](https://user-images.githubusercontent.com/1592327/139708588-32aed749-971e-4f26-a593-d4ead8d58c87.png)